packages/vue-dot/src/patterns/FormBuilder/tests/FormBuilder.spec.ts

```diff
@@ -108,6 +108,30 @@ const questionForm = {
 			multiple: true,
 			outlined: true
 		}
+	},
+	questionSelectAutocomplete: {
+		type: 'autocomplete',
+		title: 'Question select complétion automatique',
```

Comment:
```suggestion
		title: 'Question avec complétion automatique',
```
C'est mieux je pense :thinking: